src/test/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClientTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.pdf.service.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestOperations;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class PDFServiceClientTest {
+
+    private String endpointBase = "http://localhost";
+    private RestOperations restClient = mock(RestOperations.class);
+    private String sampleTemplate = "<html>Test</html>";
+    private PDFServiceClient sut;
```

Comment:
I'd rename it do `pdfServiceClient` or `client`

Reply:
Although probably depends on the existing convention

Reply:
I see your point... renaming

Reply:
@KarolJastrzebski you can also make all those field `final` while you're at it ;)